### ISSUE TITLE
Issues with CLI and tests

### ISSUE THREAD

**Issue by hellmersjl3ds** [conda/cookiecutter-conda-python#25]
Hello,
When I specify to have a CLI, I'm getting the error do **conda build**.  Do I need to configure something else prior to doing the build?
```
=================================== FAILURES ===================================
______________________________ test_cli_template _______________________________

    def test_cli_template():
>       assert cli.cli() is None

tests/test_cli.py:4:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/plpquantum/cli.py:16: in cli
    args = p.parse_args(args)
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/argparse.py:1821: in parse_args
    self.error(msg % ' '.join(argv))
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/argparse.py:2575: in error
    self.exit(2, _('%(prog)s: error: %(message)s\n') % args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = ArgumentParser(prog='pytest', usage=None, description='PLP-Quantum Computing Integration', formatter_class=<class 'argparse.HelpFormatter'>, conflict_handler='resolve', add_help=True)
status = 2, message = 'pytest: error: unrecognized arguments: tests\n'

    def exit(self, status=0, message=None):
        if message:
            self._print_message(message, _sys.stderr)
>       _sys.exit(status)
E       SystemExit: 2

../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/argparse.py:2562: SystemExit
----------------------------- Captured stderr call -----------------------------
usage: pytest [-h] [-V]
pytest: error: unrecognized arguments: tests
=========================== short test summary info ============================
FAILED tests/test_cli.py::test_cli_template - SystemExit: 2
============================== 1 failed in 0.22s ===============================
Tests failed for plpquantum-0+unknown-py_0.tar.bz2 - moving package to /home/jhellmers/anaconda3/conda-bld/broken
WARNING:conda_build.build:Tests failed for plpquantum-0+unknown-py_0.tar.bz2 - moving package to /home/jhellmers/anaconda3/conda-bld/broken
WARNING conda_build.build:tests_failed(2895): Tests failed for plpquantum-0+unknown-py_0.tar.bz2 - moving package to /home/jhellmers/anaconda3/conda-bld/broken
TESTS FAILED: plpquantum-0+unknown-py_0.tar.bz2

```

**Comment by nahid18** [comment]
Facing similar issue.

**Comment by AlbertDeFusco** [comment]
I was able to get the tests to work by running `pytest` with out any arguments from the top of output project.


Did you run the test like this?

```
pytest tests
```

**Comment by giftmischer69** [comment]
Hi, I was facing the same issue. 
When I ran `pytest [my test folder]`:
```
(dpy) D:\git\dpy>pytest tests
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov-report tests
  inifile: D:\git\dpy\setup.cfg
  rootdir: D:\git\dpy

```
When I ran `pytest`
```
(dpy) D:\git\dpy>pytest
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov-report
  inifile: D:\git\dpy\setup.cfg
  rootdir: D:\git\dpy
```

I thought I fixed it by deleting the line `--cov-report term-missing` from setup.cfg

```
(dpy) D:\git\dpy>pytest
================================================= test session starts =================================================
...
============================================ 1 passed, 2 warnings in 0.16s ============================================
```

but `pytest tests` still failed.

I finally fixed it by editing `meta.yaml` line 48:
```
  commands:
    - pytest tests
```
to
```
  commands:
    - pytest
```

TL;DR to fix it:
Edit setup.cfg -> delete `--cov-report term-missing` 
Edit meta.yaml -> change `- pytest tests` to `- pytest`

**Comment by jezdez** [comment]
Fixed in #26.
